File: 2025-1/sesion-de-clase-09/SoftProg/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/rrhh/model/Empleado.java
package pe.edu.pucp.inf30.softprog.rrhh.model;

public class Empleado extends Persona {
    private Area area;
    private CuentaUsuario cuentaUsuario;
    private String cargo;
    private double sueldo;
    
    public Area getArea() {
        return area;
    }

    public void setArea(Area area) {
        this.area = area;
    }
    
    public CuentaUsuario getCuentaUsuario() {
        return cuentaUsuario;
    }

    public void setCuentaUsuario(CuentaUsuario cuentaUsuario) {
        this.cuentaUsuario = cuentaUsuario;
    }
    
    public String getCargo() {
        return cargo;
    }

    public void setCargo(String cargo) {
        this.cargo = cargo;
    }

    public double getSueldo() {
        return sueldo;
    }

    public void setSueldo(double sueldo) {
        this.sueldo = sueldo;
    }
}

File: 2025-1/sesion-de-clase-15/SoftProgModelo/RRHH/Empleado.cs
using System;

namespace PUCP.Edu.Pe.SoftProg.Modelo.RRHH {
    public class Empleado : Persona {
        public Area Area { get; set; }
        public CuentaUsuario CuentaUsuario { get; set; }
        public string Cargo { get; set; }
        public double Sueldo { get; set; }
    }
}
